main.js

```diff
@@ -0,0 +1,135 @@
+// contentEditable = true
+
+$( document ).ready(function() {
+
+  // Create new post //
+  $("#submit").on("click", function() {
+    // Collect input from user //
+    var $name = $('#name').val();
+    var $message = $('#message').val();
+
+    // Create new div for post //
+    var $newPostDiv = $('<div class="newPostDiv"></div>');
+    var $postDiv = $('<div class="postDiv"></div>');
+    var $newPostMessage = $('<p id="post-message"></p>');
+    var $newPostName = $('<p id="post-name"></p>');
+    var $hr = $('<hr></hr>');
+    var $editDiv = $('<div class="editDiv"></div>');
+    var $editDivComments = $('<span class="comments"></span>');
+    var $editDivEdit = $('<span class="edit-post"></span>');
+    var $editDivSave = $('<span class="save-post"></span>');
+    var $editDivRemove = $('<span class="remove"></span>');
+    var $commentsDiv = $('<div class=comments-div></div>');
+    
+    $editDivComments.html('<i class="fas fa-comments"></i> Comments');
+    $editDivEdit.html('<i class="fas fa-edit"></i> Edit');
+    $editDivSave.html('<i class="fas fa-save"></i> Save');
+    $editDivRemove.html('<i class="fas fa-trash-alt"></i> Remove');
+    $newPostMessage.text($message);
+    $newPostName.text( ' - Posted By: ' + $name);
+
+    $editDiv.append($editDivComments);
+    $editDiv.append($editDivEdit);
+    $editDiv.append($editDivSave);
+    $editDiv.append($editDivRemove);
+    $postDiv.append($newPostMessage);
+    $postDiv.append($newPostName);
+    $newPostDiv.append($postDiv);
```

Comment:
We haven't taught this, so any way for now is fine, but all of this templating is a lot of variables. Normally you'd want to minimize this because any small error here could blow things up.